### PR TITLE
fix(plugin-meetings): add toggle to test integration env

### PR DESCRIPTION
**Pull request by arun3528**
# COMPLETES #< [SPARK-316745](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-316745) >

## This pull request addresses

< DESCRIBE THE CONTEXT OF THE ISSUE >

## changed done 
Currently there is no option to pass the integration token to the samples and test it out , Currently adding a small check box to check for production and unchecking will be integration env 

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [x] Internal code refactor

## The following scenarios where tested

Manually tested the sample app 

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/node_modules/samples/browser-multi-party-call/app.js [hunk 1]
```diff
@@ -32,11 +32,19 @@ let activeMeeting, webex;
...
+      hydra: 'https://apialpha.ciscospark.com/v1/'
+    }
+  };
+
   return new Promise((resolve) => {
     if (!webex) {
       // eslint-disable-next-line no-multi-assign
       webex = window.webex = Webex.init({
         config: {
+          services,
```
Comment (taymoork2) [inline packages/node_modules/samples/browser-multi-party-call/app.js:47]: You can simplify this and inline it

```suggestion
          ...(!document.getElementById("production").checked && {
	      services: {
	        discovery: {
	          u2c: "https://u2c-intb.ciscospark.com/u2c/api/v1",
	          hydra: "https://apialpha.ciscospark.com/v1/",
	        },
	      },
	    })
```

File: packages/node_modules/samples/browser-plugin-meetings/app.js [hunk 2]
```diff
@@ -70,6 +79,7 @@ function initWebex(e) {
       logger: {
         level: 'debug'
       },
+      services,
```
Comment (taymoork2) [inline packages/node_modules/samples/browser-plugin-meetings/app.js:82]: You can simplify this and inline it

```suggestion
          ...(!document.getElementById("production").checked && {
	      services: {
	        discovery: {
	          u2c: "https://u2c-intb.ciscospark.com/u2c/api/v1",
	          hydra: "https://apialpha.ciscospark.com/v1/",
	        },
	      },
	    })
```

File: packages/node_modules/samples/browser-plugin-meetings/app.js [hunk 3]
```diff
@@ -21,6 +21,7 @@ const authStatusElm = document.querySelector('#access-token-status');
 const registerElm = document.querySelector('#registration-register');
 const unregisterElm = document.querySelector('#registration-unregister');
 const registrationStatusElm = document.querySelector('#registration-status');
+const productionEnv = document.getElementById('production');
```
Comment (taymoork2) [inline packages/node_modules/samples/browser-plugin-meetings/app.js:24]: is this needed?

**Review by taymoork2 — CHANGES_REQUESTED** [review]

File: packages/node_modules/samples/browser-plugin-meetings/app.js (hunk 3 above)
Comment (arun3528) [reply packages/node_modules/samples/browser-plugin-meetings/app.js:24]: ya want to disable it and dont want to refer it twice

**Review by taymoork2 — APPROVED** [review]
FISI

**Pull request merged**